src/api/routers/config.py

```
@@ -0,0 +1,627 @@
+        else:
+            configs = manager.list_configs(ConfigType.TTS)
+            config = next((c for c in configs if c.get("id") == config_id), None)
+            if not config:
+                return {"success": False, "message": f"Configuration '{config_id}' not found"}
+            config = manager.resolve_config_env_values(config)
+
+        if not config or not config.get("base_url"):
+            return {"success": False, "message": "Configuration is incomplete (missing base_url)"}
+
+        # Test by creating client
+        client = AsyncOpenAI(
```

Ruff (0.14.11)

Local variable `client` is assigned to but never used

Remove assignment to unused variable `client`

(F841)